ios/Classes/BetterPlayerPlugin.m

```diff
@@ -405,7 +405,7 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
             [[NSNotificationCenter defaultCenter] removeObserver:self];
             [player clear];
             // This call will clear cached frame because we will return transparent frame
-
+            [player setIsPreparingDatasource:true];
```

Comment:
Please add an empty line before L408.

https://github.com/jhomlala/betterplayer/blob/master/ios/Classes/BetterPlayerPlugin.m#L299-L301
The original code has an empty line after the L300 comment.
(I think, the comment is related to `[player clear]`.